README.md

```diff
@@ -1,78 +1,120 @@
-The tools can be accessed using the GUI or the command line applications. In the picture below, the general architecture is displayed.
-
-![general_architecture](docs/assets/images/architecture.png)
-
-The tools provided are:
-* [Viewer](https://jderobot.github.io/DetectionMetrics/functionality/viewer/): view the dataset images with the annotations.
-* [Detector](https://jderobot.github.io/DetectionMetrics/functionality/detector/): run a model over a dataset and get generate a new annotated dataset.
-* [Evaluator](https://jderobot.github.io/DetectionMetrics/functionality/evaluator/): evaluate the ground truth dataset with another one and get the comparison metrics.
-* [Deployer](https://jderobot.github.io/DetectionMetrics/functionality/deployer/): run a model over different inputs like a video or webcam and generate a new annotated dataset.
-* [Converter](https://jderobot.github.io/DetectionMetrics/functionality/converter/): convert a dataset into another dataset format.
-* [Command line application (CLI)](https://jderobot.github.io/DetectionMetrics/functionality/command_line_application/): access Detection Metrics toolset through command line
-* [Detection Metrics as ROS Node](https://jderobot.github.io/DetectionMetrics/functionality/ros_node/): use Detection Metrics as a ROS Node.
-+ [Labelling](https://jderobot.github.io/DetectionMetrics/resources/gsoc_19/): add or modify labels in the datasets in runtime when running Deployer.
-
-The idea is to offer a generic infrastructure to evaluate object detection models against a dataset and compute the common statistics:
-* mAP
-* mAR
-* Mean inference time.
-
-# What's supported in Detection Metrics.
-
-| Support | Detail                                                  |
-| ------ | ------------------------------------------------------------ |
-| Supported OS  | Multiplatform using Docker                 |
-| Supported datasets  | <ul><li>COCO</li><li>ImageNet</li><li>Pascal VOC</li><li>Jderobot recorder logs</li><li>Princeton RGB dataset</li><li>Spinello dataset</li><li>Open images dataset</li></ul> |                |
-| Supported frameworks   | <ul><li>TensorFlow</li><li>Keras</li><li>PyTorch</li><li>Yolo-OpenCV</li><li>Caffe</li><li>Background substraction</li></ul>  |
-| Supported inputs in Deployer   | <ul><li>WebCamera/USB Camera</li><li>Videos</li><li>Streams from ROS</li><li>Streams from ICE</li><li>JdeRobot Recorder Logs</li></ul> |
 
+# DetectionMetrics
+
+### Project webpage [here](https://jderobot.github.io/DetectionMetrics)
+
+*DetectionMetrics* is a family of toolkits designed to unify and streamline the evaluation of perception models across different frameworks and datasets. With the release of ***DetectionMetrics v1*** we introduced a versatile suite of tools focused on object detection, supporting cross-framework evaluation and analysis. [Cite our work](#cite) if you use it in your research!
```

Comment:
Nice! We could move all the information related to v1 to the bottom of the README, with a clear reference or link at the beginning for easy navigation.

Since the paper references [DetectionMetrics](https://github.com/JdeRobot/DetectionMetrics), we should make sure to include a mention of it in the README.

Reply:
Done. Can you check the new layout?